azurerm/resource_arm_dashboard_test.go

```diff
@@ -0,0 +1,137 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+)
+
+func TestAccResourceArmDashboard_basic(t *testing.T) {
+	resourceName := "azurerm_dashboard.test"
+	ri := tf.AccRandTimeInt()
+	resourceGroupName := fmt.Sprintf("acctestRG-%d", ri)
+	config := testResourceArmDashboard_basic(ri, testLocation())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMDashboardDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDashboardExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "resource_group_name", resourceGroupName),
```

Comment:
can we remove this assertion? instead we can use an import check (see below) which confirms the state defined locally matches the code

Reply:
done